pyspecProcScripts/load_data.py

```diff
@@ -323,6 +323,11 @@ def proc_spincore_diffph_SE_v2(s, fl=None):
     return s
 
 
+def proc_spincore_general_v1(s, fl=None):
+    s = proc_spincore_generalproc_v1(s, include_tau_sub=False, direct="t")
```

Comment:
for preprocessing SC noise captures we don't want to shift the data by any tau value and the direct axis is t not t2

Reply:
The naming here is confusing... These are almost indistinguishable names at a glance 

Also, you could just use a lambda function in the look up table dictionary

Reply:
using lambda function in table dictionary now